src/main/scala/uk/gov/nationalarchives/dp/client/ContentClient.scala

```diff
@@ -79,32 +62,26 @@ object ContentClient {
           if (searchResponse.value.objectIds.isEmpty) {
             toEntities(ids)
           } else {
-            search(start + max, token, indexNames, searchResponse.value.objectIds ++ ids)
+            search(start + max, token, searchQuery, searchResponse.value.objectIds ++ ids)
           }
         )
     }
 
-    private def searchUrl(start: Int, max: Int, indexNames: ClosureResultIndexNames) = {
-      val documentStatusField = SearchField(indexNames.documentStatusName, "Closed" :: Nil)
-      val toDateTime = ZonedDateTime.now().format(DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"))
-      val reviewDateField =
-        SearchField(indexNames.reviewDateName, s"[2000-01-01T00:00:00.000Z TO $toDateTime]" :: Nil)
-      val searchQuery = SearchQuery("", List(documentStatusField, reviewDateField))
+    private def searchUri(start: Int, max: Int, searchQuery: SearchQuery): Uri = {
       val queryString = write(searchQuery)
       val queryParams = Map(
         "q" -> queryString,
         "start" -> start.toString,
         "max" -> max.toString,
-        "metadata" -> indexNames.documentStatusName
+        "metadata" -> "xip.title"
```

Comment:
Ah no, that's wrong. I've changed it to return the fields that were added in the search query. This will do for now. If it turns out later we want it to return fields that aren't in the search query then we can change it.